src/proto/index.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * This file and any referenced files were automatically generated by @osmonauts/telescope@0.10.1
+ * DO NOT MODIFY BY HAND. Instead, download the latest proto files for your chain
+ * and run the transpile command or yarn proto command to regenerate this bundle.
+ */
+
+export * from "./cosmwasm/bundle";
+export * from "./ics23/bundle";
+export * from "./cosmos_proto/bundle";
+export * from "./cosmos/bundle";
+export * from "./gogoproto/bundle";
+export * from "./google/bundle";
+export * from "./ibc/bundle";
+export * from "./tendermint/bundle";
```

Comment:
why does this file not have 
```
export * from "./persistence/bundle";
```

Reply:
interesting, it should import automatically......adding manually for now but will see whats wrong